Description: Make default of third-party cookies the most secure for users.
Author: Chad Miller <chad.miller@canonical.com>

Index: dev.trusty/src/chrome/browser/content_settings/cookie_settings.cc
===================================================================
--- dev.trusty.orig/src/chrome/browser/content_settings/cookie_settings.cc	2014-03-13 15:43:40.576304601 -0400
+++ dev.trusty/src/chrome/browser/content_settings/cookie_settings.cc	2014-03-13 15:43:40.572304459 -0400
@@ -70,7 +70,7 @@
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kBlockThirdPartyCookies,
-      false,
+      true,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 }
 
